packages/components/src/components/hds/form/text-input/field.ts

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+
+import type { ComponentLike, WithBoundArgs } from '@glint/template';
+import type { HdsFormFieldSignature } from '../field';
+import type { HdsFormTextInputBaseSignature } from './base';
+import type { HdsFormErrorSignature } from '../error';
+import type { HdsFormHelperTextSignature } from '../helper-text';
+import type { HdsFormLabelSignature } from '../label';
+import type { HdsFormVisibilityToggleSignature } from '../visibility-toggle';
+import HdsFormCharacterCountComponent from '../character-count/index.ts';
+
+interface HdsFormTextInputFieldSignature {
+  Args: Omit<HdsFormFieldSignature['Args'], 'contextualClass' | 'layout'> & {
```

Comment:
can't we just say `Omit<HdsFormFieldSignature['Args'], 'contextualClass' | 'layout'> & HdsFormTextInputBaseSignature['Args']`?

(similar to what you've done for `form/masked-input/field.ts`)

Reply:
Done in d1f958d896aa5784c69d2d7819b539854a117855